.travis.yml

```diff
@@ -30,7 +30,6 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
   allow_failures:
-    - php: nightly
```

Comment:
shouldn't we leave it in allow_failures for a while, given that it is still in alpha state (so things still can break badly)?